GetStartedGuide.md

```diff
@@ -125,8 +125,8 @@ dpct is an alias command for c2s.
 
 ## Find More
 
-* DPC++ specification:
-[https://spec.oneapi.com/versions/latest/elements/dpcpp/source/index.html](https://spec.oneapi.com/versions/latest/elements/dpcpp/source/index.html)
+* oneAPI specification:
+[https://spec.oneapi.io/releases/index.html](https://spec.oneapi.io/releases/index.html)
```

Comment:
Let's use https://spec.oneapi.io/versions/latest/index.html

Reply:
use spec.oneapi.io or spec.oneapi.com, seems spec.oneapi.com will auto redirect to spec.oneapi.io